app/Http/Controllers/Api/v1/User/Controllers/SelfController.php

```diff
@@ -36,6 +39,13 @@ public function getSelf(Request $request): JsonResponse
     {
         $user = Auth::user();
         if ($user == null) {
+            Log::error([
```

Comment:
Huh, why is a different logging method being used here? (Also different import)

Reply:
I think i forgot to update this from the old situation